pom.xml

```diff
@@ -25,7 +25,8 @@
         <dependency>
             <groupId>org.mongodb</groupId>
             <artifactId>mongo-java-driver</artifactId>
-            <version>2.13.0</version>
+            <version>3.0.0</version>
+            <scope>provided</scope>
```

Comment:
why provided? so that you can use 2.13 or 3.0? is this meant to be compatible with both?